bokeh/models/sources.py

```diff
@@ -283,6 +283,12 @@ class AjaxDataSource(RemoteSource):
     to the server. If this header is supported by the server, then only
     new data since the last request will be returned.
     """)
+    content_type = String(default='application/json', help="""
+    Set the contentType parameter in the ajax call.
+    """)
```

Comment:
Yah perhaps not worth it, especially if it can ever change or if using "custom" ones is common (I don't know if it is).